go/cmd/api-gateway/requests.go

```diff
@@ -36,18 +40,35 @@ func requestHandler() http.HandlerFunc {
 			return
 		}
 
+		userPb := &pb.User{
+			Id:       reqApproval.User.Id,
+			UserName: reqApproval.User.UserName,
+		}
+
 		// Convert the JSON request to a protobuf message
 		protoRequest := &pb.RequestApproval{
-			Type: reqApproval.Type,
-			User: &pb.User{
-				Id:       reqApproval.User.Id,
-				UserName: reqApproval.User.UserName,
-			},
+			Type:             reqApproval.Type,
```

Comment:
the comment above is now misleading
'// Convert the JSON request to a protobuf message'